oauth2/src/main/scala/com/ocadotechnology/sttp/oauth2/UserInfoProvider.scala

```diff
@@ -20,30 +19,31 @@ object UserInfoProvider {
     baseUrl: Uri,
     accessToken: String
   )(
-    implicit backend: SttpBackend[F, Nothing, NothingT]
-  ) =
-    basicRequest
-      .post(baseUrl.path(List("openid", "userinfo")))
-      .header("Authorization", s"Bearer $accessToken")
-      .response(asString)
-      .send()
+    implicit sttpBackend: SttpBackend[F, Any]
+  ): F[UserInfo] =
+    sttpBackend
+      .send {
+        basicRequest
+          .post(baseUrl.withPath("openid", "userinfo"))
```

Comment:
should we consider this?
```suggestion
          .post(baseUrl.addPath("openid", "userinfo"))
```

Reply:
It'll work, I think it's fine.

Reply:
using `withPath` to keep old behavior